apps/emails-and-messages/src/pages/api/manifest.ts

```diff
@@ -35,6 +35,10 @@ export default createManifestHandler({
       homepageUrl: "https://github.com/saleor/apps",
       supportUrl: "https://github.com/saleor/apps/discussions",
       author: "Saleor Commerce",
+      /**
+       * Requires 3.10 due to invoices event payload - in previous versions, order reference was missing
+       */
+      requiredSaleorVersion: ">=3.10 <4",
```

Comment:
```suggestion
      requiredSaleorVersion: "^3.10",
```
I do not have an opinion about which one is better. `>=3.10 <4` seems more explicit, but I see caret ranges used more often.

Reply:
fine by me, but caret == `>=` so `^3.10` == `>=3.10.0 <4.0.0-0`